src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -56,6 +56,8 @@ internal sealed partial class HttpConnection : IDisposable
         private readonly byte[] _writeBuffer;
         private int _writeOffset;
         private Exception _pendingException;
+        private readonly MaxBytesAllowed _allowedHeaderResponseBytes = new MaxBytesAllowed();
+        private readonly MaxBytesAllowed _allowChunkSizeBytes = new MaxBytesAllowed();
```

Comment:
This doesn't seem to be configurable (and I don't think we want it to be), so why isn't it something like a static readonly?

Reply:
Nevermind, I see that we are modifying the MaxBytesAllowed object, so it can't be static.